test/e2e/kubernetes/windows/windows.go

```diff
@@ -52,7 +52,7 @@ func (i *WindowsImage) GetImage(version OSVersion) string {
 
 func GetWindowsImage(testImage WindowsTestImages, version OSVersion) string {
 	httpd := WindowsImage{
-		BaseImage: "k8sprow.azurecr.io/kubernetes-e2e-test-images/httpd",
+		BaseImage: "registry.k8s.io/e2e-test-images/httpd",
```

Comment:
does that image exist and is it a windows image?
 
@jsturtevant any concern with using this image going forward?

Reply:
It is a windows image since 1.21: https://github.com/kubernetes/kubernetes/pull/77269, when I first added this test we didn't have the image in upstream.

We are good to go with this image 👍 thanks @mweibel